Eatery/Cells/MenuHeaderView.swift

```diff
@@ -12,99 +12,87 @@ import DiningStack
 @objc protocol MenuButtonsDelegate {
     func favoriteButtonPressed()
     @objc optional func shareButtonPressed()
+    @objc optional func directionsButtonPressed()
 }
 
 class MenuHeaderView: UIView {
     
     var eatery: Eatery!
     var delegate: MenuButtonsDelegate?
     var displayedDate: Date!
-    
-    var mapButtonPressed: (Void) -> Void = {}
 
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var hoursLabel: UILabel!
-    @IBOutlet weak var paymentView: UIView!
+    @IBOutlet weak var locationLabel: UILabel!
+    @IBOutlet weak var closedView: UIView!
     @IBOutlet weak var backgroundImageView: UIImageView!
-    
+    @IBOutlet var paymentImageViews: [UIImageView]!
     @IBOutlet weak var favoriteButton: UIButton!
+    @IBOutlet weak var directionsButton: UIButton!
     @IBOutlet weak var shareButton: UIButton!
-    @IBOutlet weak var mapButton: UIButton!
-    
-    override func awakeFromNib() {
-        favoriteButton.setImage(UIImage(named: "whiteStar"), for: UIControlState())
-        shareButton.setImage(UIImage(named: "shareIcon"), for: UIControlState())
-    }
     
     func setUp(_ eatery: Eatery, date: Date) {
         self.eatery = eatery
         self.displayedDate = date
-
-        // Status View
-        let eateryStatus = eatery.generateDescriptionOfCurrentState()
-        switch eateryStatus {
-        case .open(_):
-            break
-        case .closed(_):
-            break
-        }
         
-        // Payment View
-        var paymentTypeViews: [UIImageView] = []
+        backgroundColor = UIColor.groupTableViewBackground
         
-        if (eatery.paymentMethods.contains(.Swipes)) {
-            let swipeIcon = UIImageView(image: UIImage(named: "swipeIcon"))
-            paymentTypeViews.append(swipeIcon)
-        }
+        checkFavorites()
         
-        if (eatery.paymentMethods.contains(.BRB)) {
-            let brbIcon = UIImageView(image: UIImage(named: "brbIcon"))
-            paymentTypeViews.append(brbIcon)
-        }
+        var images: [UIImage] = []
         
         if (eatery.paymentMethods.contains(.Cash) || eatery.paymentMethods.contains(.CreditCard)) {
-            let cashIcon = UIImageView(image: UIImage(named: "cashIcon"))
-            paymentTypeViews.append(cashIcon)
+            images.append(#imageLiteral(resourceName: "cashIcon"))
```

Comment:
should use UIImage(named: "cashIcon") instead of #imageLiteral for consistency to avoid problems such as case sensitive image names